python/mxnet/_numpy_op_doc.py

```diff
@@ -175,4 +204,4 @@ def _np_cumsum(a, axis=None, dtype=None, out=None):
         result has the same size as `a`, and the same shape as `a` if
         `axis` is not None or `a` is a 1-d array.
     """
-    pass
+    pass
```

Comment:
Please add newline at the end of the file.

Reply:
Thx. Fixed.